<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Parallel test suite" duration-ms="876" started-at="2017-05-22T16:38:36Z" finished-at="2017-05-22T16:38:37Z">
    <groups>
      <group name="functest">
        <method signature="SimpleTest.loginCreateDeleteTest()[pri:0, instance:SimpleTest@39aeed2f]" name="loginCreateDeleteTest" class="SimpleTest"/>
      </group> <!-- functest -->
    </groups>
    <test name="Test1" duration-ms="876" started-at="2017-05-22T16:38:36Z" finished-at="2017-05-22T16:38:37Z">
      <class name="SimpleTest">
        <test-method status="FAIL" signature="loginCreateDeleteTest()[pri:0, instance:SimpleTest@39aeed2f]" name="loginCreateDeleteTest" duration-ms="860" started-at="2017-05-22T19:38:36Z" finished-at="2017-05-22T19:38:37Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at LoginPage.open(LoginPage.java:19)
	at SimpleTest.loginCreateDeleteTest(SimpleTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCreateDeleteTest -->
      </class> <!-- SimpleTest -->
    </test> <!-- Test1 -->
  </suite> <!-- Parallel test suite -->
</testng-results>
